src/main/groovy/org/mockito/release/notes/vcs/GitContributionsProvider.java

```diff
@@ -27,7 +27,7 @@ public ContributionSet getContributionsBetween(String fromRev, String toRev) {
 
         DefaultContributionSet contributions = new DefaultContributionSet();
         for (Commit commit : commits) {
-            if (!ignoredCommit.isTrue(commit)) {
+            if (ignoredCommit.isTrue(commit)) {
```

Comment:
You forgot about renaming the field - name "ignoredCommit" suggests that we add ignored commits to contributions